### PR TITLE
LPS-191341 Set a configurable GraphQL depth limit

### PR DESCRIPTION
**Pull request by 4lejandrito**
From https://github.com/brianchandotcom/liferay-portal/pull/141543#issue-1919166247.

I fixed SF and we are waiting for Brian to reply on slack to figure out what is wrong with it.

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
ci:test:sf

**Comment by jalber786** [comment]
ci:forward

**Comment by jalber786** [comment]
ci:test:sf

**Comment by jalber786** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -8016,7 +8016,8 @@ headings-line-height=Headings Line Height
 headless=Headless
 headless-action-key=Headless Action Key
 headless-action-key-help=This key is used to display the action if the user has the permission for it by checking if the key is present in the data item's actions array.
-headless-api-configuration-name=Headless API
```
Comment (4lejandrito) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:8019]: Hey @jalber786, brian asked to have both keys `company` and non company with the same value.

Comment (jalber786) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:8019]: But isn't it what It's written? I mean both of the endpoint ones have the exact same value, the one that changes is the GraphQL one

**Comment by jalber786** [comment]
ci:forward:force

**Pull request closed without merging**